README.md

```diff
@@ -1 +1,457 @@
+| --------| -------------- | ---------------------------------------------------- | -------- |
+| `awv`   | `"0.1.0"`      | AWAKE message version                                | Yes      |
+| `type`  | `"awake/init"` | Signal which step of AWAKE this payload is for       | Yes      |
+| `did`   |                | The DID of the Requestor this is intended for        | Yes      |
+| `caps`  |                | Capabilities that the Responder MUST provide         | Yes      |
+
+#### 3.2.3.1 JSON Example
+
+``` javascript
+{
+  "awv": "0.1.0",
+  "type": "awake/init"
+  "did": "did:key:z4MXj1wBzi9jUstyPMS4jQqB6KdJaiatPkAtVtGc6bQEQEEsKTic4G7Rou3iBf9vPmT5dbkm9qsZsuVNjq8HCuW1w24nhBFGkRE4cd2Uf2tfrB3N7h4mnyPp1BF3ZttHTYv3DLUPi1zMdkULiow3M1GfXkoC6DoxDUm1jmN6GBj22SjVsr6dxezRVQc7aj9TxE7JLbMH1wh5X3kA58H3DFW8rnYMakFGbca5CB2Jf6CnGQZmL7o5uJAdTwXfy2iiiyPxXEGerMhHwhjTA1mKYobyk2CpeEcmvynADfNZ5MBvcCS7m3XkFCMNUYBS9NQ3fze6vMSUPsNa6GVYmKx2x6JrdEjCk3qRMMmyjnjCMfR4pXbRMZa3i",
+  "caps": [
+    {
+      "with": "mailto:me@example.com",
+      "can": "msg/send"
+    },
+    {
+      "with": "dns:example.com",
+      "can": "crud/update"
+    },
+    {
+      "with": "as:did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp:*",
+      "can": "*"
+    }
+  ]
+}
+```
+
+## 3.3 Responder Establishes Point-to-Point Session
+
+**NOTE: The Responder is not yet trusted at this step, and MUST be treated as a possible impersonator or PITM**
+
+```
+Requestor                  Responder
+    ⋮                          ⋮
+    │       Authorization      │ (3a)
+    │       & Session Key      │ (3b)
+    │◄─────────────────────────┤
+    ⋮                          ⋮
+```
+
+In this step, the Responder MUST prove that they have access to the requested resources, and MUST set the input key material (IKM) that will be used to start a protected point-to-point connection. This is used to establish trust in the capabilities of the Responder, but MUST NOT actually delegating anything.
+
+The temporary RSA key from the previous step MUST be exclusively used for exchanging a 256-bit AES "session key". RSA is both slow and can only hold a limited number of bytes, so using it to encrypt the payloads of the rest of the session is infeasable.
+
+The payload contains two encryption layers, and signature: the RSA envelope, the AES envelope, and the AES key signed by the Responder.
+
+```
+          Payload
+
+        ┌───RSA───┐
+        │         │
+        │ AES-GCM │
+        │    │    │
+        └────┼────┘
+             │
+             │
+             ▼
+┌─────────AES-GCM────────┐
+│                        │
+│  ┌───────UCAN───────┐  │
+│  │                  │  │
+│  │  iss: Responder  │  │
+│  │  aud: TempDID    │  │
+│  │  fct: AES-GCM    │  │
+│  │  att: []         │  │
+│  │  prf: ...        │  │
+│  │                  │  │
+│  └──────────────────┘  │
+│                        │
+└────────────────────────┘
+```
+
```

Comment:
Is there a concern here that some attacker continuously responds to messages before the actual responder?

I'm specifically thinking about regenerating the temp DID here.
- Alice sends out request which is seen by Bob and Eve.
- Eve quickly responds with a bad UCAN
- Alice throws out tempDID & tries again
- repeat until Bob gets a faster internet connection 😛

Reply:
Oh yeah good point, I should add text that this is nonblocking 👍 (i.e. multiple connections allowed)